src/material-examples/icon-overview/icon-overview-example.html

```diff
@@ -1 +1 @@
-<mat-icon>home</mat-icon>
+<mat-icon aria-hidden="false" aria-label="Example home icon">home</mat-icon>
```

Comment:
I believe that the right thing here is to still keep it as `aria-hidden`. cc @jelbourn

Reply:
This one is somewhat tricky. The icon by itself doesn't actually *mean* anything since it's just an example. The two options would be to do this (hidden="false" with a label), or to add a separate visibly-hidden span that describes the example. 

I think this PR is probably fine as is- it's at least _better_ than what it was before.